etcd-cluster.yaml

```diff
@@ -53,6 +53,10 @@ Resources:
         FromPort: 2379
         ToPort: 2380
         CidrIp: 0.0.0.0/0
+      - IpProtocol: tcp
+        FromPort: 9100
+        ToPort: 9100
+        CidrIp: 0.0.0.0/0
```

Comment:
Shouldn't that be only ZMON or other monitoring tool being used?  I.e. would be a manual task to update SG after deploying?